scene/resources/texture.cpp

```diff
@@ -1324,17 +1324,19 @@ void CubeMap::_bind_methods() {
 
 	ClassDB::bind_method(D_METHOD("get_width"), &CubeMap::get_width);
 	ClassDB::bind_method(D_METHOD("get_height"), &CubeMap::get_height);
-	//ClassDB::bind_method(D_METHOD("get_rid"),&CubeMap::get_rid);
 	ClassDB::bind_method(D_METHOD("set_flags", "flags"), &CubeMap::set_flags);
 	ClassDB::bind_method(D_METHOD("get_flags"), &CubeMap::get_flags);
-
 	ClassDB::bind_method(D_METHOD("set_side", "side", "image"), &CubeMap::set_side);
```

Comment:
this could be properties too if im not mistaken. see `ADD_PROPERTYI` in control.cpp for example

Reply:
Looking at it it could, but looking at Control.cpp you have to then make a separate method for each item. For GUI anchors it probably makes more sense than a cube map. In this particular case I would recommend against it, or have both a method and properties.

For instance, instead of:
``` cpp
for (int i = 0; i < 6; i++)
{
    cubemap_sides[i] = get_side(i);
    // do work on cubemap_sides[i]
    set_side(i, cubemap_sides[i]);
}
```

You have to:
``` cpp
cubemap_sides[0] = top_side;
cubemap_sides[1] = bottom_side;
cubemap_sides[2] = left_side;
cubemap_sides[3] = right_side;
cubemap_sides[4] = front_side;
cubemap_sides[5] = back_side;

for ( int i = 0; i < 6; i++ )
{
     // do work on cubemap_sides[i]
}

top_side = cubemap_sides[0];
bottom_side = cubemap_sides[1];
left_side = cubemap_sides[2];
right_side = cubemap_sides[3];
front_side = cubemap_sides[4];
back_side = cubemap_sides[5];
```

It would also make writing the code more difficult when you want to do thing like update one side per frame for performance reasons.

What do you think?

Reply:
Thinking about it again, I think it would be good to have both. I'll be out for a little bit and will implement the PROPERTYI.